### ISSUE TITLE
Power button does not lock or unlock

### ISSUE THREAD

**Issue by harmtemolder** [damianmqr/A9PatchingService#26]
I patched the most recent `lineage-21.0-20241118-UNOFFICIAL-arm64_bvS.img.gz` from <https://sourceforge.net/projects/andyyan-gsi/files/>, which seemed successful until it boots: the power button does not lock or unlock the screen. When I flashed the original image, without patching, it worked OK. How can I help debug this?

Edit: Tested with v1.4.1 and the power button works as expected, so must be something in v3.5.0, right?

**Comment by jaromeyer** [comment]
@harmtemolder Were you able to even reach the setup screen? I tried patching the same GSI and the screen just went blank after the initial boot screen. I was able to connect via adb but unfortunately scrcpy didn't work.

**Comment by harmtemolder** [comment]
@jaromeyer Yes, flashing and setup went OK. Everything else seemed to work as well, except for the power button
